develop.sh

```diff
@@ -46,6 +52,10 @@ elif [ "$1" == "npm" ]; then
 	npm_install
 	exit
 
+elif [ "$1" == "spark" ]; then shift
+	invoke_docker_compose_spark "$@"
```

Comment:
I wonder if we could use `exec` in this script and every place that we call docker-compose: https://askubuntu.com/a/819960. We should be able to add it in the `invoke_` commands
It would prevent us from having to use exit in each block, and puts control back in the hands of docker-compose